css/sass/spacing-classes.scss

```diff
@@ -0,0 +1,100 @@
+/**
+ * @margin-and-padding.scss
+ * global classes for margin and padding
+ * naming order follows the property shorthand:
+ * • top • right • bottom • left
+ * m or p (margin or padding)
+ * side  (top • right • bottom • left)
+ * amount and unit (px • em • rem)
+ * example generated classes:
+ *   .m-trbl--1px, .m-t--1em, m-b--1rem
```

Comment:
The benefit is adding less classes to the html. While you can certainly chain it if you so choose, I think having options for all groups seems easier.

`.class.m-t--1rem.m-r--1rem.m-b--1rem.m-l--1rem` vs `.class.m-trbl-1rem` 

Can and be used in a combo `.class.m-trb--5rem.m-l--10rem`